capstone/capdb/models.py

```diff
@@ -915,6 +921,14 @@ class Meta:
     def save(self, *args, **kwargs):
         if self.in_scope != self.get_in_scope():
             self.in_scope = not self.in_scope
+
+        # Add a custom footer message to instance if redactions or elisions exist but no text is provided
+        if not self.custom_footer_message and (self.no_index_redacted or self.no_index_elided):
```

Comment:
I would do this in the view so it's easier to change later.

Reply:
If we do this in the view, we're going to have to do another case lookup. I'm trying to keep case lookups to a minimum.... Do you think that's needless to worry about?